daemon/historydaemon.cpp

```diff
@@ -1250,6 +1251,7 @@ void HistoryDaemon::onMessageSent(const Tp::TextChannelPtr textChannel, const Tp
     event[History::FieldEventId] = eventId;
     event[History::FieldSenderId] = "self";
     event[History::FieldTimestamp] = QDateTime::currentDateTime().toString("yyyy-MM-ddTHH:mm:ss.zzz"); // FIXME: check why message.sent() is empty
+    event[History::FieldSentTime] = QDateTime::currentDateTime().toString("yyyy-MM-ddTHH:mm:ss.zzz");
```

Comment:
Does this invalidate the comment directly above it?

`// FIXME: check why message.sent() is empty`

Reply:
yes